pkg/signals/shutdown.go

```diff
@@ -0,0 +1,53 @@
+package signals
+
+import (
+	"context"
+	"time"
+
+	"get.porter.sh/porter/pkg/tracing"
+	"github.com/spf13/viper"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
```

Comment:
naming here could be like `otel` `oteltrace` so we know what sdk we are playing w/